### PR TITLE
Sept - Update anchor links

### LINKED ISSUE #10843: Broken Header Links Found
Specific header links in the [Workflow Run](https://github.com/cloudflare/cloudflare-docs/actions/runs/6286835454).

Generally, we expect you to address at least 10-15 links before closing this out.

### PR DESCRIPTION
**Pull request by kodster28**
closes #10843

### CODE REVIEW

**Review by irvinebroque — APPROVED** [review]
Some of these were mine, thanks for catching Kody

**Comment by kodster28** [comment]
Gotcha. If you ever want, you can run our anchor link auditing against a specific branch (note that this will create an issue in our repo if it finds anything).

1. https://github.com/cloudflare/cloudflare-docs/actions/workflows/anchor-link-audit.yml
2. 
Screenshot 2023-09-25 at 9 25 49 AM

File: content/queues/learning/consumer-concurrency.md [hunk 1]
```diff
@@ -103,8 +103,8 @@ When multiple consumer Workers are invoked, each Worker invocation incurs [durat
 * If you intend to process all messages written to a queue, _the effective overall cost is the same_, even with concurrency enabled.
 * Enabling concurrency simply brings those costs forward, and can help prevent messages from reaching the [message retention limit](/queues/platform/limits/).
 
-Billing for consumers follows the [Workers unbound usage model](/workers/platform/pricing/#usage-models) meaning a developer is billed for the request and the duration of the request. 
+Billing for consumers follows the [Workers unbound usage model](/workers/platform/pricing/#workers) meaning a developer is billed for the request and the duration of the request.
```
Comment (RebeccaTamachiro) [inline content/queues/learning/consumer-concurrency.md]:
```suggestion
Billing for consumers follows the [Workers unbound usage model](/workers/platform/pricing/#unbound-usage-model) meaning a developer is billed for the request and the duration of the request. 
```

File: content/workers/observability/metrics-and-analytics.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ The CPU Time per execution chart shows historical CPU time data broken down into
 
 The Duration per execution chart shows historical [duration](/workers/platform/limits/#duration) per Worker execution. The data is broken down into relevant quantiles, similar to the CPU time chart. Learn more about [interpreting quantiles](https://www.statisticshowto.com/quantile-definition-find-easy-steps/). Understanding duration on your Worker is especially useful when you are intending to do a significant amount of computation on the Worker itself.
 
-Workers on the [Bundled Usage Model](/workers/platform/pricing/#usage-models) may have high durations, even with a 50 ms CPU time limit, if they are running many network-bound operations like `fetch` requests, and waiting on responses.
+Workers on the [Bundled Usage Model](/workers/platform/pricing/#workers) may have high durations, even with a 50 ms CPU time limit, if they are running many network-bound operations like `fetch` requests, and waiting on responses.
```
Comment (RebeccaTamachiro) [inline content/workers/observability/metrics-and-analytics.md]:
```suggestion
Workers on the [Bundled Usage Model](/workers/platform/pricing/#bundled-usage-model) may have high durations, even with a 50 ms CPU time limit, if they are running many network-bound operations like `fetch` requests, and waiting on responses.
```

File: content/workers/platform/limits.md [hunk 3]
```diff
@@ -64,7 +64,7 @@ Cloudflare does not enforce response limits, but cache limits for [Cloudflare's
 
 {{<table-wrap>}}
 
-| Feature                     | Free                                       | [Bundled Usage Model](/workers/platform/pricing/#usage-models) | [Unbound Usage Model](/workers/platform/pricing/#usage-models)|
+| Feature                     | Free                                       | [Bundled Usage Model](/workers/platform/pricing/#workers) | [Unbound Usage Model](/workers/platform/pricing/#workers)|
```
Comment (RebeccaTamachiro) [inline content/workers/platform/limits.md]:
```suggestion
| Feature                     | Free                                       | [Bundled Usage Model](/workers/platform/pricing/#bundled-usage-model) | [Unbound Usage Model](/workers/platform/pricing/#unbound-usage-model)|
```

File: content/workers/runtime-apis/cache.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ The Cache API is available globally but the contents of the cache do not replica
 
 {{<Aside type="warning" header="Tiered caching">}}
 
-The `cache.put` method is not compatible with tiered caching. Refer to [Cache API](/workers/learning/how-the-cache-works/#cache-api) for more information. To perform tiered caching, use the [fetch API](/workers/learning/how-the-cache-works/#interacting-with-the-cloudflare-cache).
+The `cache.put` method is not compatible with tiered caching. Refer to [Cache API](/workers/learning/how-the-cache-works/#cache-api-limits) for more information. To perform tiered caching, use the [fetch API](/workers/learning/how-the-cache-works/#interacting-with-the-cloudflare-cache).
```
Comment (RebeccaTamachiro) [inline content/workers/runtime-apis/cache.md]:
```suggestion
The `cache.put` method is not compatible with tiered caching. Refer to [Cache API](/workers/learning/how-the-cache-works/#cache-api) for more information. To perform tiered caching, use the [fetch API](/workers/learning/how-the-cache-works/#interacting-with-the-cloudflare-cache).
```

Comment (RebeccaTamachiro) [reply content/workers/runtime-apis/cache.md]: I think that one was actually working before. It's just very similar to the other fix re: `/workers/platform/limits/#cache-api-limits` but this one is for `workers/learning/how-the-cache-works`

**Review by RebeccaTamachiro — APPROVED** [review]

**Pull request merged**